service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureDisabledTest.java

```diff
@@ -38,7 +38,7 @@
 @Disabled("Until we get the postgres connection leaks addressed")
 @Tag("connected")
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
-public class AzureDisabledTest extends BaseConnectedTest {
```

Comment:
This one seems like a `BaseConnectedTest`- it's validating that a feature is respected, not using Azure functionality

Reply:
sure, aligns with the test gag. changed